### PR TITLE
Add siboot

### PR DESCRIPTION
**Pull request by zetadeceleste**

### CODE REVIEW

File: games/s.yaml [hunk 1]
```diff
@@ -401,6 +401,22 @@
   type: remake
   updated: 2018-06-15
 
+  name: 'Trust & Betrayal: The Legacy of Siboot'
```
Comment (cxong) [inline games/s.yaml:404]:
```suggestion
- name: 'Trust & Betrayal: The Legacy of Siboot'
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/s.yaml (hunk 1 above)
Comment (zetadeceleste) [reply games/s.yaml:404]: I'm sorry, I don't get the change.

Comment (cxong) [reply games/s.yaml:404]: this is yaml syntax for creating a new array item; without it this would be part of the previous item

Comment (zetadeceleste) [reply games/s.yaml]: Ok! gotcha!

File: games/s.yaml [hunk 2]
```diff
@@ -401,6 +401,22 @@
...
+    website: 'https://siboot.org/'
+  wikipedia: 'https://en.wikipedia.org/wiki/Trust_%26_Betrayal:_The_Legacy_of_Siboot'
+  platform:
+  - DOS
+  meta:
+    genre:
+    - Educational
+    - Simulation
+    - Strategy
+  theme:
```
[the remark this answers is not in the document]
Comment (zetadeceleste) [reply games/s.yaml]: Now it's ok?

File: games/s.yaml [hunk 3]
```diff
@@ -401,6 +401,22 @@
   type: remake
   updated: 2018-06-15
 
+- name: 'Trust & Betrayal: The Legacy of Siboot'
```
Comment (cxong) [inline games/s.yaml:404]: this looks like the wrong file

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/s.yaml (hunk 3 above)
Comment (zetadeceleste) [reply games/s.yaml:404]: I added at originals too, but i don't know if it's ok

Comment (cxong) [reply games/s.yaml:404]: the files under games/ are for clones
the originals go under originals/
the two types of files have different schemas, so this stuff doesn't belong here

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/s.yaml (hunk 3 above)
Comment (zetadeceleste) [reply games/s.yaml:404]: I checked this syntax with https://osgameclones.com/add_original.html as you recommend and pushed,
would you tell me please if it's correct?

Comment (zetadeceleste) [reply games/s.yaml:404]: So, you are saying instead of adding this game at /games I should move it to /originals?
It's that correct?

Comment (cxong) [reply games/s.yaml:404]: correct; please refer to the schema and readme

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/s.yaml (hunk 3 above)
Comment (zetadeceleste) [reply games/s.yaml]: done, please let me know! thanks

**Review by cxong — APPROVED** [review]

**Pull request merged**